rest/attachment_test.go

```diff
@@ -2355,6 +2355,8 @@ func TestPushUnknownAttachmentAsStub(t *testing.T) {
 }
 
 func TestMinRevPosWorkToAvoidUnnecessaryProveAttachment(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyCRUD, base.KeySync, base.KeyChanges, base.KeyHTTP)
```

Comment:
Add `KeySGTest` maybe for debugging blip client?

Reply:
I'm gonna close this PR - since your advice to run `-count=500 -race` worked to repro locally - don't need the additional logging merged for Jenkins runs.